action.yml

```diff
@@ -5,17 +5,20 @@ branding:
   icon: 'edit-3'
   color: 'blue'
 inputs:
-  dhall-binary:
-    description: 'Binary of dhall-haskell to use'
+  dhall-json-binary:
+    description: 'Binary of dhall-json to use'
```

Comment:
We're definitely going to need to make a new release (since anyone using the old version will break).